channeldb/invoices.go

```diff
@@ -11,7 +11,7 @@ import (
 
 	"github.com/btcsuite/btcd/wire"
 	"github.com/coreos/bbolt"
-	"github.com/lightningnetwork/lnd/lntypes"
+	"github.com/lightningnetwork/lnd/lnhash"
```

Comment:
`lnhash` is what I had initially, but after comments I changed it to `lntypes`.

Reply:
removed import move